### PR TITLE
Issue - 1358 Argument error when converting token updated_at using to…

### PR DESCRIPTION
**Pull request by saichander17**
Fix for #1358 .

### CODE REVIEW

File: app/models/devise_token_auth/concerns/user.rb [hunk 1]
```diff
@@ -158,7 +158,7 @@ def create_new_auth_token(client = nil)
     token = create_token(
       client: client,
       last_token: tokens.fetch(client, {})['token'],
-      updated_at: now
+      updated_at: now.to_s(:rfc822)
```
Comment (MaicolBen) [inline app/models/devise_token_auth/concerns/user.rb]: wouldn't this break other's people defaults?

Comment (saichander17) [reply app/models/devise_token_auth/concerns/user.rb]: @MaicolBen , Should it not be used like updated_at.to_time. Even in the whole code of this particular gem, whenever it needs to be compared with time, to_time was always used.

**Comment by MaicolBen** [comment]
I will rollback this change, it breaks #1423 which was requested by a lot of people (compared with this one that is only you)

**Comment by MaicolBen** [comment]
If you want this change back, please add a new PR with a test that verifies the bug. Maybe you have to read `Time::DATE_FORMATS[:default]`

**Pull request merged**